front/lib/multi_regions/region_lookup_client.ts

```diff
@@ -0,0 +1,78 @@
+import { EnvironmentConfig, isDevelopment } from "@dust-tt/types";
+
+import type {
+  UserLookupRequestBodyType,
+  UserLookupResponse,
+} from "@app/pages/api/lookup/[resource]";
+
+const CLUSTER_REGIONS = ["local", "europe-west1", "us-central1"] as const;
+type ClusterRegionType = (typeof CLUSTER_REGIONS)[number];
+
+const REGIONS_TO_URLS: Map<ClusterRegionType, string> = new Map([
+  ["local", "http://localhost:3000"],
+  ["europe-west1", "https://eu.dust.tt"],
+  ["us-central1", "https://dust.tt"],
+]);
+
+type Resource = "user";
+
+function isCurrentRegion(region: ClusterRegionType): boolean {
```

Comment:
Could we add this as a method in `config`?

Reply:
But don't mind leaving it as is, was a nit 🤝

Reply:
> because IIRC EnvironmentConfig is only used in */config.ts files

Fair, I re-introduced a config file and moved stuff in it :)